python/lsst/ts/planning/tool/zephyr_interface.py

```diff
@@ -303,6 +409,79 @@ async def get_test_executions(self, test_cycle_key, max_results=20):
 
         return values
 
+    async def get_user_name(self, account_id):
+        """
+        Get the user name based on a JSON object containing "self"
+        and "accountId" keys.
+
+        Parameters
+        ----------
+        account_id : int
+            The account ID of the user.
+
+        Returns
+        -------
+        str
+            The user name.
+        """
+        endpoint = "user"
+        url = self.jira_base_url + endpoint
+
+        if self.jira_api_token is None:
+            raise ValueError("JIRA API token is not set")
+
+        query_parameters = {
+            "accountId": account_id,
+        }
+
+        async with aiohttp.ClientSession(
+            auth=BasicAuth(f"{self.jira_username}@lsst.org", self.jira_api_token)
+        ) as session:
+            async with session.get(url, params=query_parameters) as response:
+                response_text = await response.text()
+                if response.status == 200:
+                    user_details = await response.json()
+                    self.log.debug(
+                        f"Token is working fine. User display name: {user_details['displayName']}"
+                    )
+                else:
+                    raise aiohttp.ClientError(
+                        f"Failed to authenticate. Status code: {response.status} {response_text}"
+                    )
+        return user_details["displayName"]
+
+    async def get_project(self, project_id):
+        """
+        Query the project based on a JSON object containing "self" and
+        "id" keys.
```

Comment:
This docstring should be updated as now you are usign directly the `project_id` as param instead of the `json_object`.